driver/xorwow_skipahead_generator.hpp

```diff
@@ -53,7 +53,7 @@
 #define XORWOW_JUMP_LOG2_DEV 1
 #define XORWOW_SEQUENCE_JUMP_LOG2 67
 
-unsigned int xorwow_next(prngStates* cur_state)
+static unsigned int xorwow_next(prngStates* cur_state)
```

Comment:
This is to avoid "duplicate definition" linking error.

Reply:
@atamazov 
I would make it either `inline` or `extern` with appropriate definition.
`inline` is an official way to solve `duplicate definition` problem in C++ - it will inline or pick just one definition based on heuristics. `static` is a kind of workaround - it hides the name from external visibility and does not allow to reuse it.

Reply:
@CAHEK7 yep, but I tried to keep diff as small as possible (without much success ugh)

Reply:
That is strange, because `inline` is exactly what should be used in this case. Let me check.

Reply:
For example `random.hpp`
```
inline glibc_gen& get_prng()
{
    static thread_local glibc_gen gen{get_default_seed()};
    return gen;
}
```
works perfectly. As well as the other functions.

Reply:
@CAHEK7 inline actually works here, fixed.